keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2763,6 +2763,17 @@ Test links:
   appropriate and useful).
 - Allowing time for feedback
 - Scalability and performance analysis done and published
+- Define kubelet restart, node bootstrap/restart, driver runtime, and
+  upgrade/downgrade failure modes
+  - Implement kubelet, [DRA kubeletplugin
```

Comment:
can you fix the indentations here? I think just a flat list is fine

@klueska any additional itmes you want to add?

Reply:
fixed formatting in 29dbede